src/server/repositories/UrlRepository.ts

```diff
@@ -23,9 +24,13 @@ import { SearchResultsSortOrder } from '../../shared/search'
 import { urlSearchVector } from '../models/search'
 import { DirectoryQueryConditions } from '../modules/directory'
 import { extractShortUrl, sanitiseQuery } from '../util/parse'
+import { Tag, TagType } from '../models/tag'
+import { TagRepositoryInterface } from './interfaces/TagRepositoryInterface'
 
 const { Public, Private } = FileVisibility
 
+export const tagSeparator = ';'
```

Comment:
nit, can we use capital letters for the constant

export const TAG_SEPARATOR = ';'

Reply:
done